src/main/php/com/handlebarsjs/BlockHelpers.class.php

```diff
@@ -13,6 +13,8 @@ static function __static() {
     self::$byName['unless']= XPClass::forName('com.handlebarsjs.UnlessBlockHelper');
     self::$byName['with']= XPClass::forName('com.handlebarsjs.WithBlockHelper');
     self::$byName['each']= XPClass::forName('com.handlebarsjs.EachBlockHelper');
+    self::$byName['>']= XPClass::forName('com.handlebarsjs.PartialBlockHelper');
+    self::$byName['*inline']= XPClass::forName('com.handlebarsjs.InlinePartialHelper');
```

Comment:
This pull request does not support generic decorators as e.g.

```handlebars
{{#grid data}}
  {{#*column "Column 1"}}{{foo}}{{/column}}
  {{#*column "Column 2"}}{{bar}}{{/column}}
{{/grid}}
```

...just `*inline` as a special case!